Makefile

```diff
@@ -133,6 +134,19 @@ test-sim-backwards-compatible: ## Test uploading a file with lastest release (je
 
 ##@ Build
 
+.PHONY: storagenode-web
+storagenode-web:
+	# install npm dependencies
+	cd web/storagenode; npm ci
+	# build web assets
+	rm -rf web/storagenode/dist
+	cd web/storagenode; npm run build
+	# embed web assets into go
+	go-bindata -prefix web/storagenode/dist/ -fs -o storagenode/console/consoleassets/bindata.go -pkg consoleassets web/storagenode/dist/...
```

Comment:
Any reason for not using `cmd/internal/asset`?

Reply:
i forgot about it

Reply:
are you using `cmd/internal/asset` now? go-bindata is still mentioned in places.

Reply:
no, and i haven't had time to switch to it still. i would like to propose that switching to cmd/internal/asset be done as a separate, later pr at this point. jens would like this into the next release branch. this works.

Reply:
reason switching to cmd/internal/asset is non completely trivial: cmd/internal/asset does not support selectively adding subfolders to the root, and we only want to add "static" and "dist" out of web/storagenode, which go-bindata supports